addons/l10n_ar/models/account_move.py

```diff
@@ -167,4 +167,14 @@ def _reverse_moves(self, default_values_list=None, cancel=False):
                 'l10n_ar_afip_service_start': move.l10n_ar_afip_service_start,
                 'l10n_ar_afip_service_end': move.l10n_ar_afip_service_end,
             })
+
+            # Required for option "modify" of account.move.reversal
+            refund_document = self.env.context.get('refund_l10n_latam_document_type_id')
```

Comment:
Too much hacking.

Reply:
@jco-odoo @zaoral 
If we add the _prepare_default_reversal method suggested by @jco-odoo I believe this is no longer needed and can be done nicer.